### PR TITLE
Feature/dropdown

### PR DESCRIPTION
**Pull request by hjalmers**

### CODE REVIEW

**Comment by sebopensource** [comment]
👋 Thanks for creating a pull request! 🚀 Checkout the storybook we created for it [here](https://sebgroup.github.io/green/pr-218/chlorophyll/).

File: apps/angular-lib-dev/src/app/dropdown/dropdown.module.ts [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import { NgModule } from '@angular/core';
+import { CommonModule } from '@angular/common';
+
+import { DropdownRoutingModule } from './dropdown-routing.module';
+import { DropdownComponent } from './dropdown.component';
+import { ReactiveFormsModule } from '@angular/forms'
+// eslint-disable-next-line
+import { GreenDropdownModule } from '@sebgroup/green-angular'
```
Comment (JohanObrink) [inline apps/angular-lib-dev/src/app/dropdown/dropdown.module.ts:8]: Green seems redundant in the name. It's a DropdownModule from `green-angular`. That should be enough.

Comment (JohanObrink) [reply apps/angular-lib-dev/src/app/dropdown/dropdown.module.ts:8]: Otherwise it seems fine

**Pull request merged**